apps/gitness/src/components/FileExplorer.tsx

```diff
@@ -27,6 +27,13 @@ const sortEntriesByType = (entries: OpenapiContentInfo[]): OpenapiContentInfo[]
   })
 }
 
+/**
+ * TODO: This code was migrated from V2.
+ *
+ * @param selectedBranch
+ * @param repoDetails
+ * @constructor
```

Comment:
These don't really describe this component. This isn't a constructor for a class, so `@constructor` doesn't apply and it takes a single param (props) with 2 items.
```suggestion
```